tasks/filter-team-data/team_data_filterer.rb

```diff
@@ -49,13 +49,14 @@ def write_json(data, path)
     def projects_by_name(projects)
       results = {}
       projects.each do |project|
-        name = project['name']
+        name = project['name'].downcase
         results[name] = project
       end
       results
     end
 
     def build_target(target, p_by_name)
+      target['name'].downcase!
```

Comment:
Can we duplicate the Hash rather than modifying it? Don't want any unintended side effects from modifying arguments.